google/cloud/security/common/data_access/sql_queries/snapshot_cycles_sql.py

```diff
@@ -24,6 +24,7 @@
         `complete_time` datetime DEFAULT NULL,
         `status` enum('SUCCESS','RUNNING','FAILURE',
                       'PARTIAL_SUCCESS','TIMEOUT') DEFAULT NULL,
+        `has_all_data` bool DEFAULT NULL,
```

Comment:
Gotcha, then I think we have a need to rename both fields:

`pipeline_execution_status`
`pipeline_data_status`

The later should be a boolean. We should also define these and describe the enum in our docs for `pipeline_execution_status`